src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,16 @@
-export const UserInfo = () => {};
+import { User } from '../../types/User';
+
+export type Props = {
```

Comment:
```suggestion
export interface Props = {
```
why need to export props type from the component?